src/transformers/models/layoutlmv3/modeling_layoutlmv3.py

```diff
@@ -1384,6 +1430,19 @@ def forward(
         )
 
 
+def _cogview_attention(attention_scores: torch.Tensor, alpha: Union[int, float] = 32) -> torch.Tensor:
```

Comment:
we dont'really need a separate function but if we keep it place it at the top please